db_test.go

```diff
@@ -34,6 +33,71 @@ var (
 	err error
 )
 
+func assertErr(t *testing.T, err error, expectErr error) {
+	if expectErr != nil {
+		require.Equal(t, expectErr, err)
+	} else {
+		require.NoError(t, err)
+	}
+}
+
+func removeDir(dir string) {
+	if err := os.RemoveAll(dir); err != nil {
+		panic(err)
+	}
+}
+
+func runNutsDBTest(t *testing.T, opts *Options, test func(t *testing.T, db *DB)) {
+	if opts == nil {
+		opts = new(Options)
+		*opts = DefaultOptions
+	}
+	if opts.Dir == "" {
+		opts.Dir = "/tmp/nutsdb-test"
```

Comment:
Maybe making this field a const value will be better. It can be used by other test functions.

Reply:
Okay